lib/rspec/its.rb

```diff
@@ -123,6 +141,18 @@ def is_expected
         end
         alias_method :are_expected, :is_expected
 
+        def will(matcher=nil, message=nil)
+          raise ArgumentError, "`will` only supports block expectations" \
+            unless matcher.supports_block_expectations?
```

Comment:
I think we'd prefer:
```
unless matcher.supports_block_expectations?
  raise ArgumentError, "`will` only supports block expectations"
end
```

Rather than a line contraction

Reply:
👍